### PR TITLE
Create docs.md

### PR DESCRIPTION
**Pull request by Boutainaelatbaoui**
brief documentation in Arabic about the translation process of the project in Crowdin.

### CODE REVIEW

**Comment by Boutainaelatbaoui** [comment]
Hello @BatoolMM, I just submitted a PR. Could you please give me your feedback?

File: docs/docs.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+# # توثيق عملية ترجمة المشروع في كراودن
```
Comment (BatoolMM) [inline docs/docs.md]:
```suggestion
## توثيق عملية ترجمة المشروع في كراودن
```

Comment (BatoolMM) [reply docs/docs.md]: Crowdin is supposed to be written:
(Crowdin) كراودن

File: docs/docs.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+يعمل [مجتمع البيانات المفتوحة بالمملكة العربية السعودية](https://form.jotform.com/220362171812446) مع [منهج تورينج](https://the-turing-way.netlify.app/welcome.html) لتعريب مواردها التعليمية المفتوحة وإتاحتها للمتعلمين المتحدثين باللغة العربية، وذلك عن طريق ترجمة وتدقيق فصول محددة منه باستخدام [كراودن](https://crowdin.com/)،
```
Comment (BatoolMM) [inline docs/docs.md]:
```suggestion
يعمل [مجتمع البيانات المفتوحة بالمملكة العربية السعودية](https://form.jotform.com/220362171812446) مع [منهج تورينج](https://the-turing-way.netlify.app/welcome.html) لتعريب مواردها التعليمية المفتوحة وإتاحتها للمتعلمين المتحدثين باللغة العربية، وذلك عن طريق ترجمة وتدقيق فصول محددة منه باستخدام [كراودن](https://crowdin.com/).
```

File: docs/docs.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+**كراودن** هو عبارة عن منصة لإدارة التوطين تُساعد الشركات على ترجمة برامجها. وتُستخدم هذه المنصة كنظام إدارة الترجمة لتقديم المساهمات على نحو سلس، حيث تُوفّر ذاكرة الترجمة، وقاموس المصطلحات،والترجمة الآلية، وغيرها من الميزات،
+
+### عملية ترجمة المشروع في كراودن
+
+* بعد القيام بجميع الخطوات الضروريّة للانتقال إلى صفحة [الترجمة العربية](https://turingway.crowdin.com/u/projects/1/l/ar) (إنشاء حساب على كراودن، والانضمام إلى [المشروع](https://turingway.crowdin.com/u/))،
+
+![screenshot2](https://user-images.githubusercontent.com/56041419/162965483-2dcd2200-5da3-43e8-a7f0-5c1c760a3775.PNG)
+
+يجب اختيار ملف واحد للعمل عليه من فصل `بحث قابل لإعادة الإنتاج`، ثم النقر على **الانتقال إلى المحرر**،
```
Comment (BatoolMM) [inline docs/docs.md]:
```suggestion

 يجب اختيار ملف واحد للعمل عليه من فصل `بحث قابل لإعادة الإنتاج`، ثم النقر على **الانتقال إلى المحرر**. 
- ```

File: docs/docs.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+بعد ذلك تُفتح مباشرة صفحة **محرر كراودن**، حيث يمكننا تقسيمها إلى ثلاثة أقسام رئيسية:
+
+* **القسم الأول**: يتكون من قائمة سلاسل، وهي عناصر نصية تكون إما كلمات أو تعبيرات أو جمل فردية، ويتوفر هذا القسم على خيارات لإدارة عرض هذه العناصر، ومعاينة الترجمة، وغيرها من الميزات،
+* **القسم الثاني**: منطقة العمل الرئيسية، حيث تظهر الجمل وترجماتها المقترحة التي يُمكن الموافقة عليها أو تصحيحها أو تغييرها بالكامل،
+* **القسم الثالث**: يسمح قسم التعليقات بمناقشة وطرح الأسئلة ذات الصلة بالمشروع، وذلك عبر استخدام "@" ثم اسم المستخدم لتوجيه الرسالة إلى الشخص المحدد،
+
+أما القائمة الرئيسية في الزاوية اليسرى العلوية، فتُستخدم للتبديل بين الملفات أو تغيير لغة الترجمة أو تبديل العرض أو الحصول على المساعدة أو الخروج من المحرر،
+
+
+المزيد من المعلومات حول كيفية استخدام محرر كراودن [هنا](https://support.crowdin.com/online-editor/)،
```
Comment (BatoolMM) [inline docs/docs.md]:
```suggestion
المزيد من المعلومات حول كيفية استخدام محرر كراودن [هنا](https://support.crowdin.com/online-editor/).
```

File: docs/docs.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+
+![screenshot10](https://user-images.githubusercontent.com/56041419/163281963-20c63d26-f95c-4825-ac04-aae1f36a8191.PNG)
+
+وقبل البدء في الترجمة يُفضَّل تصفح المحتوى عبر الملفات بدلاً من القائمة، وذلك من أجل فهم سياق النص وترجمته على أساس ذلك (كما توضّح الصورة أدناه يمكن تغيير طريقة العرض بالنقر على الرمز المحاط بالدائرة)،
```
Comment (BatoolMM) [inline docs/docs.md]:
```suggestion
وقبل البدء في الترجمة يُفضَّل تصفح المحتوى عبر الملفات بدلاً من القائمة، وذلك من أجل فهم سياق النص وترجمته على أساس ذلك (كما توضّح الصورة أدناه يمكن تغيير طريقة العرض بالنقر على الرمز المحاط بالدائرة).
```

**Review by BatoolMM — CHANGES_REQUESTED** [review]

**Comment by BatoolMM** [comment]
Very comprehensive review - thank you for the PR. Can you fix the punctuation at the end of each **paragraph**, it should be a full stop, not a "comma". Also, can you write two sentences at the end summarising your own experience? did you find Crowdin easy or hard,  was the part you translated about "open data" interesting?

**Comment by Boutainaelatbaoui** [comment]
I just updated the file and made all Changes requested. Thanks so much for your feedback @BatoolMM .

**Review by BatoolMM — APPROVED** [review]

**Comment by BatoolMM** [comment]
@Boutainaelatbaoui - It looks awesome, I  like how you elaborated on the Crowdin Editor. Please don't forget to submit your final contribution through [this link](https://www.outreachy.org/outreachy-may-2022-internship-round/communities/open-science-community-saudi-arabia-oscsa/localisation-of-open-educational-resources-of-open/contributions/) include the steps you followed & what is the timeline to go through all the tasks - and what you anticipate to get out of this project. Please let me know if anything isn't clear!

**Comment by Boutainaelatbaoui** [comment]
@BatoolMM Could you please help me? I'm unsure which link to put in my final contribution.
This link:
https://github.com/Open-Science-Community-Saudi-Arabia/localisation/blob/main/docs/docs.md
Or this:
https://github.com/Open-Science-Community-Saudi-Arabia/localisation/pull/6.
And thank you very much for your guidance and support.

**Comment by BatoolMM** [comment]
You should add [this link](https://github.com/Open-Science-Community-Saudi-Arabia/localisation/pull/6) to your final contribution! Let me know if you have more questions!

**Comment by Boutainaelatbaoui** [comment]
Hello @BatoolMM, I submitted the final application, and I want to make sure that I don't have to provide an additional community-specific questions.

**Comment by Boutainaelatbaoui** [comment]
Hello @BatoolMM, I submitted the final application, and I want to make sure that I don't have to provide additional community-specific questions in the final application.

**Comment by BatoolMM** [comment]
Yes, you don't have to - thank you @Boutainaelatbaoui for submitting your final application!

**Comment by Boutainaelatbaoui** [comment]
Thank you so much @BatoolMM  for your answer, and I am so grateful for your help and support.

**Pull request merged**